### PR TITLE
inherit workspace properties

### PR DESCRIPTION
**Pull request by NicolaLS**
using the new rust. if we need to change a common property we only need to do it in the main cargo toml now

### CODE REVIEW

File: client/client-lib/Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,14 @@
 [package]
 name = "mint-client"
-version = "0.1.0"
-authors = ["The Fedimint Developers"]
-edition = "2021"
 description = "mint-client provides a library for sending transactions to the federation."
-license = "MIT"
+version.workspace = true
```
Comment (dpc) [inline client/client-lib/Cargo.toml:4]: Wait, what? We need to do it this way for every field in every package? That's so meh...

Comment (dpc) [reply client/client-lib/Cargo.toml:4]: Looks like it's true... :shrug:

**Review by dpc — APPROVED** [review]

File: client/client-lib/Cargo.toml (hunk 1 above)
Comment (NicolaLS) [reply client/client-lib/Cargo.toml:4]: yeah.. would be nice if you could inherit a whole group or smth.. still better like this for changing stuff in the future

**Comment by NicolaLS** [comment]
blocked on updating crane. waiting for https://github.com/ipetkov/crane/issues/76#issuecomment-1296040525

**Comment by justinmoon** [comment]
I'm going to close this for now. Doesn't seem to be much interest in it. Can always re-open later.

**Pull request closed without merging**